Week02/lesson04/homework/Layout Challenge/Three Column/styles.css

```diff
@@ -0,0 +1,20 @@
+* {
+  -webkit-box-sizing: border-box;
+}
+
+  .top {
+  background-color: #ffff99;
+  border: 2px solid #ffff00;
+  margin: 2px;
+  height: 150px;
+}
+
+.middle {
+  display: inline-block;
+  height: 350px;
+  width: 20%;
```

Comment:
Don't blame your for making it 20% and saving yourself the headache you no doubt experienced when doing the two-column layout! Explanation coming, but you grasped all the most important layout concepts so 🌟 for that! 😄